spec/reek/tree_walker_spec.rb

```diff
@@ -5,9 +5,9 @@
 # Dummy repository to inject into TreeWalker in order to count statements in
 # all contexts.
 class TestSmellRepository
-  attr_reader :num_statements
+  attr_accessor :num_statements
   def examine(context)
-    @num_statements = context.num_statements
+    self.num_statements = context.num_statements
```

Comment:
Yep, and I didn't do so because that statement made me realize that @mvz had already done that. :laughing: